### PR TITLE
Bug 2091825 - if a host in infra env is in "disconnected" status, there is no indication of it in the list view

### PR DESCRIPTION
**Pull request by ammont82**
Related to https://bugzilla.redhat.com/show_bug.cgi?id=2091825

### CODE REVIEW

File: frontend/src/routes/Infrastructure/InfraEnvironments/InfraEnvironmentsPage.tsx [hunk 1]
```diff
@@ -188,9 +188,12 @@ const InfraEnvsTable: React.FC<InfraEnvsTableProps> = ({ infraEnvs, agents }) =>
                             )
                             const errorAgents = infraAgents.filter((a) => getAgentStatus(a).status.key === 'error')
                             const warningAgents = infraAgents.filter((a) =>
-                                ['pending-for-input', 'insufficient', 'insufficient-unbound'].includes(
-                                    getAgentStatus(a).status.key
-                                )
+                                [
+                                    'pending-for-input',
+                                    'insufficient',
+                                    'insufficient-unbound',
```
Comment (jtomasek) [inline frontend/src/routes/Infrastructure/InfraEnvironments/InfraEnvironmentsPage.tsx:194]:
```suggestion
                                    'insufficient-unbound',
                                    'disconnected',
```

**Review by jtomasek — CHANGES_REQUESTED** [review]

File: frontend/src/routes/Infrastructure/InfraEnvironments/InfraEnvironmentsPage.tsx (hunk 1 above)
Comment (ammont82) [reply frontend/src/routes/Infrastructure/InfraEnvironments/InfraEnvironmentsPage.tsx:194]: changes done!

**Review by jtomasek — APPROVED** [review]
/lgtm

**Pull request merged**